payments/mobile-pay/other-features.md

```diff
@@ -564,7 +564,7 @@ not yet reversed.
 
 ```mermaid
 sequenceDiagram
-  Merchant->>PayEx: POST <creditcard reversal>
+  Merchant->>PayEx: POST <mobilepay reversal>
```

Comment:
```suggestion
  Merchant->>PayEx: POST <MobilePay Online reversal>
```

Reply:
Since the `reversal` applies directly to the API, which is called `mobilepay`, I don't agree with this change.